docs/bis/leftNav.md

```diff
@@ -118,6 +159,16 @@ $('#guide-list').slideToggle()
 }
 });
 </script>
+<script>
+$("#domain-schemas").append(function () {
```

Comment:
Please change this to `$("#domain\\ schemas")`. Escaping the space will allow jQuery to select the "domain schemas" id and add a chevron.

Reply:
This is not correct.  HTML Element ids cannot contain spaces, and multiple ids are not supported. We should fix this on the bemetalsmith end, if we're creating an invalid id.

Reply:
@diegoalexdiaz We will be releasing a new version of BeMetalsmith soon. It fixes the heading ID issues.